examples/uptime_sla/migrations/4_uptime_sla.js

```diff
@@ -1,22 +1,40 @@
-let chainlinkDeployer = require("../chainlink_deployer.js");
-let UptimeSLA = artifacts.require("./UptimeSLA.sol");
-let Oracle = artifacts.require("../../../solidity/contracts/Oracle.sol");
-let LINK = artifacts.require("../../../solidity/contracts/LinkToken.sol");
+const clmigration = require("../clmigration.js");
+const request = require("request-promise");
+const UptimeSLA = artifacts.require("./UptimeSLA.sol");
+const Oracle = artifacts.require("../../../solidity/contracts/Oracle.sol");
+const LINK = artifacts.require("../../../solidity/contracts/LinkToken.sol");
 
-module.exports = function(truffleDeployer) {
+let url = "http://chainlink:twochains@localhost:6688/v2/specs";
```

Comment:
Should we be using: `"http://"+process.env.USERNAME+":"+process.env.PASSWORD+"@localhost:6688/v2/specs"`

Reply:
I was hoping to move away from hard-coding defaults and using whatever system's values are set for $USERNAME and $PASSWORD instead. However, I don't mind going back to chainlink:twochains.